BLSQ/iaso
# POLIO-1388: Add the analytics code from the account to the calendar page of polio

Explain what problem this PR is resolving
- There is a need to track embedded calendar usage
- This PR is about to add analytics script into embedded calendar page
Related JIRA tickets : [POLIO-1388](https://bluesquare.atlassian.net/browse/POLIO-1388)

## Self proofreading checklist


## Changes
- Add account_analytics_script into calendarSerializer
- Get distinct analytics script from campaign account_analytics_script
- Append the analytics script to head tag

## How to test
- Add an analytics script in an account
- Go to the embedded calendar
- inspect the code and check if the script is in the head tag

## Print screen / video
[Screencast from 2024-01-17 15-18-36.webm](https://github.com/BLSQ/iaso/assets/19631540/49008d98-71ef-40f6-ad72-c2e7218d01fa)



[POLIO-1388]: https://bluesquare.atlassian.net/browse/POLIO-1388?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
